src/Http/Controllers/HomeController.php

```diff
@@ -10,7 +10,7 @@ class HomeController extends AbstractController
     public function getIndex()
     {
         // Only redirect to status/dashboard if we have no referer
-        if (Auth::check() && (request()->server('HTTP_REFERER') !== '') && (url()->previous() === url()->current())) {
+        if (Auth::check() && (request()->server('HTTP_REFERER') !== '')) {
```

Comment:
I have this weird feeling that second condition should be changed to `url()->previous() !== url()->current()` instead of being completely removed.

Reply:
I think `url()->previous()` defaults to `url()->current()` if you have no previous page (i.e. no referrer), instead of an expected `null`. But it acted wonky sometimes. I need to test this.

Reply:
Sorry for the noise, I was worried about cyclic redirection (which this part of condition seemed to safeguard against) but I forgot that this code in in a home controller, which is a different URL than redirection target.